.gitignore

```diff
@@ -30,3 +30,7 @@
 .phpunit.result.cache
 .phpunit.cache
 ###< phpunit/phpunit ###
+
+###> twigstan/twigstan ###
+.twigstan
+###< twigstan/twigstan ###
```

Comment:
Should be added to the bundle recipe @ruudk no?

Reply:
I guess, but there is no recipe yet 😅

Reply:
Ho sorry, I have forgotten, it is not a bundle yet  (and maybe it is better without).